queries/loans.ql

```diff
@@ -0,0 +1,71 @@
+query MyQuery {
```

Comment:
Is the queries folder required? It looks like it might be a scratchpad folder for queries run during manual testing.

Reply:
that is correct -- I should move it somewhere else